examples/twitter_support/main.py

```diff
@@ -0,0 +1,98 @@
+from dotenv import load_dotenv
+load_dotenv()
+from monkey_patch.monkey import Monkey as monkey
+from pydantic import BaseModel
+from typing import Literal
+
+class Response(BaseModel):
+    """
+    Response object, where the response attribute is the response sent to the customer
+    requires_ticket is a boolean indicating whether the incoming tweet was a question or a direct issue
+    """
+    requires_ticket: bool 
+    response: str 
+
+class SupportTicket(BaseModel):
+    """
+    Support ticket object, where the name attribute is customers name,
+    issue is a brief description of the issue customer had
+    urgency conveys how urgently the team should respond to the issue
+    """
+    name: str
+    issue: str
+    urgency: Literal["low", "medium", "high"]
+
+@monkey.patch
+def classify_and_respond(tweet_text: str) -> Response:
+    """
+    Respond to the customer support tweet text empathetically and nicely. 
+    Convey that you care about the issue and if the problem was a direct issue that the support team should fix or a question, the team will respond to it. 
+    """
+
+@monkey.align
+def align_respond():
+    input_tweet_1 = "Laia Johnson: I really like the new shovel but the handle broke after 2 days of use. Can I get a replacement?"
+    assert classify_and_respond(input_tweet_1) == Response(requires_ticket=True, response="Hi, we are sorry to hear that. We will get back to you with a replacement as soon as possible, can you send us your order nr?")
```

Comment:
Can you lint this please @MartBakler? Ideally creating new lines between the kwargs so we don't line wrap.

Reply:
Updated